Tasks/AppCenterTest/appcentertest.ts

```diff
@@ -63,22 +63,22 @@ function getCliPath(): string {
         return userDefinedPath;
     }
 
-    let systemPath = tl.which('mobile-center', false);
+    let systemPath = tl.which('appcenter', false);
     if (systemPath) {
         return systemPath;
     }
 
-    // On Windows (Hosted Agent) we attempt to use the bundled mobile-center cli as user doesn't have any
-    // chance to install the CLI themselves.  On Mac the bundled mobile-center does not work due to some
+    // On Windows (Hosted Agent) we attempt to use the bundled appcenter cli as user doesn't have any
+    // chance to install the CLI themselves.  On Mac the bundled appcenter does not work due to some
```

Comment:
Optional: Does the npm bundled appcenter work on Mac now?

Reply:
No, we still bundle the CLI only for windows. It is because the task zip we ship is built on windows.